tests/userTest.js

```diff
@@ -6,17 +6,17 @@ describe('Main: Currently testing user management,', function () {
   // PACT setup
   const {
     ocsMeta,
-    capabilitiesGETRequestValidAuth,
-    GETRequestToCloudUserEndpoint,
-    createAUser,
-    deleteAUser,
-    createAUserWithGroupMembership,
+    getCapabilitiesInteraction,
+    getCurrentUserInformationInteraction,
+    createAUserInteraction,
+    deleteUserInteraction,
+    createAUserWithGroupMembershipInteraction,
```

Comment:
```suggestion
    createUserInteraction,
    deleteUserInteraction,
    createUserWithGroupMembershipInteraction,
```
You removed "A" from `deleteAUser` - could do the same for the `createAUser*`